src/services/sig.py

```
@@ -135,6 +167,10 @@ def get_by_id(self, id: int) -> SIG:
             raise HTTPException(404, detail="해당 id의 시그/피그가 없습니다")
         return sig
 
+    def get_sig_response_by_id(self, id: int) -> SigResponse:
```

Ruff (0.15.5)

[error] Function argument `id` is shadowing a Python builtin

(A002)